config/initializers/assets.rb

```diff
@@ -9,11 +9,7 @@
 # from version 4: https://github.com/rails/sprockets/blob/58cca17aa447fcee17703e4ab4dbfaab630e7ed4/UPGRADING.md
 # Rails.application.config.assets.precompile += %w[govuk_publishing_components_manifest.js]
```

Comment:
Small nitpick: non-blocking, we should be able to remove this line to remove a bit of duplication, essentially putting this block of code back to how it was before supporting both `dartsass-rails` and `sassc-rails` - https://github.com/alphagov/govuk_publishing_components/pull/3726/commits/0898868e3aa2921d1b6cb6b73633592eb2ad6e64#diff-52a4487c125f78c29e9b0cec0fbae1a2330507596d5f7cdd32c3d52886c84d4aL10

Reply:
Have changed, does that look correct?

Reply:
Looks good to me, thanks 👍